polygeist/lib/Dialect/Polygeist/Transforms/LoopInternalization.cpp

```diff
@@ -9,6 +9,49 @@
 // This pass tiles perfect loop nests to 'prefetch' memory accesses in shared
 // local memory.
 //
+// Before optimization:
+//   size_t global_id = nditem.get_global_id(0)
+//   for (size_t k = 0; k < N; ++k)
+//     A[k];
+//
+// After optimization (pseudocode):
+//   // Obtain shared local memory for all candidate memory access in a kernel
+//   needed for this optimization.
```

Comment:
```suggestion
//   // Obtain shared local memory for all candidate memory accesses in a kernel
//   needed for this optimization.
```

Reply:
Remove double  '//'

Reply:
The double '//' is intentional to indicate that it is a comment in the example.

Reply:
Changed `access` to `accesses`.